GUI/deprecated/from_itchio.py

```diff
@@ -0,0 +1,49 @@
+import requests
+import json
+from threading import Thread, Lock
+import platform
+import os
+
+lock = Lock()
+
+class DownloadThread(Thread):
+	def __init__(self, get_request, chunks, path, name):
+		Thread.__init__(self)
+		self.get_request = get_request
+		self.chunks = chunks
+		self.path = path
+		if not self.path.endswith("/"):
+			if platform.system().lower()=="windows":
```

Comment:
Does windows allow using / as a path separator?

Reply:
no, but I fixed that in the last commit IIRC.